src/storage/src/hummock/event_handler/uploader.rs

```diff
@@ -116,14 +117,17 @@ impl UploadingTask {
     /// Poll the result of the uploading task
     fn poll_result(&mut self, cx: &mut Context<'_>) -> Poll<HummockResult<StagingSstableInfo>> {
         Poll::Ready(match ready!(self.join_handle.poll_unpin(cx)) {
-            Ok(task_result) => task_result.map(|ssts| {
-                StagingSstableInfo::new(
-                    ssts,
-                    self.task_info.epochs.clone(),
-                    self.task_info.imm_ids.clone(),
-                    self.task_info.task_size,
-                )
-            }),
+            Ok(task_result) => task_result
+                .inspect(|_| info!("upload task finish {:?}", self.task_info))
```

Comment:
The compaction_group_index in task_info may have a lot of information, which may cause the log size of the ci to exceed the limit